sdk.conf

```diff
@@ -1 +1 @@
-osgeo4a_version=20201231
+osgeo4a_version=ndk_sideload
```

Comment:
Did you want to update that to an official release tag before merging?

Reply:
```suggestion
osgeo4a_version=20210106
```